### DISCUSSION TITLE
[Unstable] add submarine function is broken

### DISCUSSION THREAD

**Discussion by jcx1991911** [FakeFishGames/Barotrauma#14346] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

https://github.com/user-attachments/assets/81cf9bda-d00b-47c7-a407-30194839d48d

it dont work (

### Reproduction steps

in the video

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/8ec8cd61d50b9eae42e6c50949a362a2c126a8e3 (commit in our private repo). Fix coming up in the next unstable build!
